### PR TITLE
create model User with devise, and model Terrace

### PR DESCRIPTION
**Pull request by Scream10**
Hi, can you review my code please ?

Best nini's ever ! (thank you)

### CODE REVIEW

**Comment by lililestrade** [comment]
NON

File: db/migrate/20200217143415_create_terraces.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class CreateTerraces < ActiveRecord::Migration[5.2]
+  def change
+    create_table :terraces do |t|
+      t.string :name
+      t.text :description
+      t.string :address
+      t.integer :longitude
+      t.integer :lattitude
```
Comment (lililestrade) [inline db/migrate/20200217143415_create_terraces.rb]: checker si integer ou string ..

Comment (Scream10) [reply db/migrate/20200217143415_create_terraces.rb]: I change it into float, should work now 🎉

Comment (lililestrade) [reply db/migrate/20200217143415_create_terraces.rb]: amazing

Comment (stephaniesassier) [reply db/migrate/20200217143415_create_terraces.rb]: au top !

**Pull request merged**